hermes-console/static/js/console/topic/TopicRepository.js

```diff
@@ -20,7 +20,7 @@ repository.factory('TopicRepository', ['DiscoveryService', '$resource', '$locati
         }
 
         function wrapSchemaRemove(topic, promise) {
-            if (topicConfig.removeSchema && topic.contentType == 'AVRO') {
+            if (topicConfig.removeSchema && topic.contentType === 'AVRO') {
```

Comment:
...is there a case where `==` is wrong when comparing to 'AVRO'? :> 
+1 of course for this practice, but I'm just wondering.